packages/okta-react-native/ios/OktaSdkBridge.podspec

```diff
@@ -0,0 +1,22 @@
+
+Pod::Spec.new do |s|
+  s.name         = "OktaSdkBridge"
+  s.version      = "1.0.0"
+  s.summary      = "ROktaSdkBridgeNNewlibrary"
```

Comment:
Just `ROktaSdkBridge`

BTW, should it contain `RN` prefix, so it shows that it is react native lib. Something like RNOktaOidcBridge

Reply:
Would that change affect all the module names? My current module name is named to OktaSdkBridge